pkg/apis/kubermatic/v1/datacenter.go

```diff
@@ -90,6 +90,66 @@ func IsProviderSupported(name string) bool {
 	return false
 }
 
+// +kubebuilder:validation:Enum="";Healthy;Unhealthy;Invalid;Terminating;Paused
+
+type SeedPhase string
+
+// These are the valid phases of a seed.
+const (
+	// SeedHealthyPhase means the seed is reachable and was successfully reconciled.
+	SeedHealthyPhase SeedPhase = "Healthy"
+
+	// SeedUnhealthyPhase means the KKP resources on the seed cluster could not be
+	// successfully reconciled.
+	SeedUnhealthyPhase SeedPhase = "Unhealthy"
+
+	// SeedInvalidPhase means the seed kubeconfig is defunct.
+	SeedInvalidPhase SeedPhase = "Invalid"
+
+	// SeedTerminatingPhase means the seed is currently being deleted.
+	SeedTerminatingPhase SeedPhase = "Terminating"
+
+	// SeedPausedPhase means the seed is not being reconciled because if SkipReconciling
+	// annotation is set.
+	SeedPausedPhase SeedPhase = "Paused"
+)
+
+// +kubebuilder:validation:Enum="";SeedConditionResourcesReconciled;SeedConditionValidKubeconfig;
+
+// SeedConditionType is used to indicate the type of a seed condition. For all condition
+// types, the `true` value must indicate success. All condition types must be registered
+// within the `AllSeedConditionTypes` variable.
```

Comment:
```suggestion
// SeedConditionType is used to indicate the type of a seed condition. For all condition
// types, `true` value will indicate success. All condition types must be registered
// within the `AllSeedConditionTypes` variable.
```

Reply:
Removing "the" would turn this into broken English, no?

Reply:
I guess you could drop the "value" after "`true`" and that will make it valid English again, but yeah the suggestion does sound a bit broken.